content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -121,7 +116,6 @@ brew update && brew install kops
 {{% /tab %}}
 {{< /tabs >}}
 
-
 ### (2/5) Create a route53 domain for your cluster
 
 kops uses DNS for discovery, both inside the cluster and outside, so that you can reach the kubernetes API server
```

Comment:
```suggestion
kops uses DNS for discovery, both inside the cluster and outside, so that you can reach the kubernetes API server.
```

Reply:
I don't understand why two commas in one sentence